rootfs/scheduler/k8s.py

```diff
@@ -305,6 +379,25 @@ def scale(self, name, image, command, **kwargs):
             self._scale_app(name, old_replicas, app_name)
             raise RuntimeError('{} (Scale): {}'.format(name, e))
 
+    def _create_namespace(self, app_name):
+        url = self._api("/namespaces")
+        data = {
+            "kind": "Namespace",
+            "apiVersion": self.apiversion,
+            "metadata": {
+                "name": app_name
+                }
+            }
+        resp = self.session.post(url, json=data)
+        if not resp.status_code == 201:
+            error(resp, "create Namespace {}".format(app_name))
+
+    def _check_status(self, resp, app_name):
+        if resp.status_code == 404:
+            self._create_namespace(app_name)
+        elif resp.status_code != 200:
```

Comment:
_create_rc has more than 12 conditional statements which is not passing flake tests. using unhealhty wont solve the problem here, So came up with a separate function.